Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.14
+
+
+WORKDIR /linkerd2-conformance
+
+COPY bin/ bin/
+COPY sonobuoy/ sonobuoy/
+COPY tests/ tests/
+COPY utils/ utils/
+COPY go.mod .
+COPY go.sum .
+# Build the test binary
+RUN go test -i ./tests/...
```

Comment:
Are we just checking for syntax errors here?

Reply:
The `-i` flag installs the dependencies and compiles our tests for any errors, but does not run them. This was done so that when users are running the tests on Sonouoy, the pod does is only focussed towards running the tests and not install any dependencies.
I forgot to remove `# Build the test binary` comment; it is from when the Dockerfile was a lot different.